### PR TITLE
Turn `createSchemaField` into a method on the solr Client class.

### PR DESCRIPTION
**Pull request by rudolfbyker**

### CODE REVIEW

**Comment by rudolfbyker** [comment]
Hold up, found an error.

**Comment by kibertoad** [comment]
Guess we need a new test then :)

File: lib/solr.ts [hunk 1]
```diff
@@ -880,6 +880,40 @@ export class Client {
...
+    const payload = {
+      'add-field': {
+        name: fieldName,
+        type: fieldType,
+        multiValued: false,
+        stored: true,
+      },
+    };
+
+    const params = {
```
Comment (kibertoad) [inline lib/solr.ts:894]: can we type this? is this Request options or something from http module?

**Comment by rudolfbyker** [comment]
No, it's just that the method I moved had the hard-coded assumption built in that it would only be used in tests...

File: lib/solr.ts [hunk 2]
```diff
@@ -880,6 +880,40 @@ export class Client {
   ping(callback: CallbackFn) {
     return this.get(this.ADMIN_PING_HANDLER, callback);
   }
+
+  createSchemaField(fieldName, fieldType, cb) {
```
Comment (kibertoad) [inline lib/solr.ts:884]: can we add types?

Comment (rudolfbyker) [reply lib/solr.ts:884]: Yes, thanks for reminding me.

File: lib/solr.ts (hunk 1 above)
Comment (rudolfbyker) [reply lib/solr.ts:894]: I'm not exactly sure. It's was only used in tests. I'll have a look.

Comment (rudolfbyker) [reply lib/solr.ts:894]: This is the argument expected by postJSON, which I will remove / replace / properly type in a next PR.

Remember, I'm only **moving** a method in this PR, not rewriting or even refactoring it.

Comment (rudolfbyker) [reply lib/solr.ts:894]: It's the same as here https://github.com/lbdremy/solr-node-client/blob/master/lib/solr.ts#L594

**Review by kibertoad — APPROVED** [review]

**Pull request merged**